packages/odo-expandable/demos/styles/demo.css

```diff
@@ -2,11 +2,30 @@
   display: none;
 }
 
-.odo-expandable__target--open {
+.expandable__target.odo-expandable__target--open {
   display: block;
 }
 
 .expandable__trigger {
   width: 100%;
 }
 
+.expandable__trigger--animated {
+  width: 100%;
+}
+
+.expandable__target--animated {
+  width: 100%;
+  overflow: hidden;
+  transition: height .3s ease-out;
```

Comment:
Does it make sense to add a small css file with the required css to make the animated version work? It's probably just `overflow: hidden` and `transition: height` needed right?

Reply:
I was going back and forth about it but I think so. I'll add one in!